academy/xl-cosmjs/first-steps.md

```diff
@@ -37,7 +37,7 @@ const runAll = async(): Promise<void> => {
 runAll()
 ```
 
-To execute, this Typescript file needs to be [compiled into Javascript](https://github.com/deus-labs/cosmjs-template/blob/main/tsconfig.json#L3) before [being interpreted](https://github.com/deus-labs/cosmjs-template/blob/main/package.json#L25) by NodeJs. Add it as a run target in `package.json`:
+To execute, this Typescript file needs to be compiled into Javascript before being interpreted by NodeJs. As you can see, we've already added this for you as a run target in `package.json`:
```

Comment:
```suggestion
To execute, this Typescript file needs to be compiled into Javascript before being interpreted by NodeJs. Add this as a run target in `package.json`:
```
To keep it in line with the style of other calls to action with the result in a code block and the link to the done work.
@nooomski